File: Beecrowd/Beginner/2203 - ~~/2311 - Diving/Python/2311_Diving.py
divers = int(input())

for _ in range(divers):
    name = input()
    difficulty = float(input())
    grades = [float(x) for x in input().split()]

    grades.sort()

    total = sum(grades[1:6])

    print(f"{name} {total * difficulty:.2f}")

File: Beecrowd/Beginner/2203 - ~~/2311 - Diving/C++/2311_Diving.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int divers;
    cin >> divers;

    while (divers--) {
        string name;
        cin >> name;

        double difficulty;
        cin >> difficulty;

        double grades[7];
        for (int i = 0; i < 7; i++) {
            cin >> grades[i];
        }

        sort(grades, grades + 7);

        double total = 0;
        for (int i = 1; i < 6; i++) {
            total += grades[i];
        }

        cout << name << " " << fixed << setprecision(2) << total * difficulty << endl;
    }

    return 0;
}